renderdoc/driver/d3d12/d3d12_device.cpp

```diff
@@ -39,6 +39,17 @@
 
 WRAPPED_POOL_INST(WrappedID3D12Device);
 
+void WrappedID3D12Device::RemoveQueue(WrappedID3D12CommandQueue* queue)
+{
+  auto it = std::remove_if(m_Queues.begin(), m_Queues.end(),
+                           [queue](const WrappedID3D12CommandQueue* q) {return q == queue; });
+
+  if (it != m_Queues.end())
+    m_Queues.erase(it);
+  else
+    RDCERR("Unknown wrapped queue %llu being remvoed", queue);
```

Comment:
`%p` is a more appropriate printf specifier here, because it is used for printing a pointer, but `%llu` is uint64_t sized - so it would read garbage on a 32-bit build.

Reply:
Thanks. I copied this line from somewhere else in the code, should have been more careful.